hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestEndpoints.java

```diff
@@ -366,25 +370,19 @@ public void setUp() throws Exception {
     } catch (Exception ex) {
       Assertions.fail(ex.getMessage());
     }
-
     // Write Data to OM
-
     // A sample volume (sampleVol) and a bucket (bucketOne) is already created
     // in AbstractOMMetadataManagerTest.
     // Create a new volume and bucket and then write keys to the bucket.
     String volumeKey = reconOMMetadataManager.getVolumeKey("sampleVol2");
     OmVolumeArgs args =
         OmVolumeArgs.newBuilder()
-            .setVolume("sampleVol2")
-            .setAdminName("TestUser")
-            .setOwnerName("TestUser")
-            .build();
+            .setVolume("sampleVol2").setAdminName("TestUser")
+            .setOwnerName("TestUser").build();
```

Comment:
It's usually a convention to have one setter per line to use a builder. No need to change these.